components/eamxx/src/share/io/eamxx_io_utils.cpp

```diff
@@ -231,4 +233,48 @@ create_diagnostic (const std::string& diag_field_name,
   return diag;
 }
 
+std::pair<std::string, std::string>
+parse_field_alias (const std::string& field_spec)
+{
+  const std::string delimiter = ":=";
```

Comment:
The first part of this fcn could be replaced with

```c++
  auto tokens = ekat::split(field_spec,":=");
  EKAT_REQUIRE_MSG (tokens.size()<=2,
    "Error! Invalid field alias specification: '" + field_spec + "'\n"
    "Multiple ':=' tokens found.\n");

  auto alias = tokens[0];
  auto field_name = tokens[1];
```

Reply:
Yeah, if tokens.size()==1, we should just return `{ekat::trim(tokens[0]), ekat::trim(tokens[0]})`.

Reply:
In the version I posted in my comment, it should throw an exception since `tokens.size()>2`.